unstructured/ingest/pipeline/interfaces.py

```diff
@@ -71,6 +71,8 @@ def __call__(self, iterable: t.Optional[t.Iterable[t.Any]] = None) -> t.Any:
             else:
                 self.result = self.run()
         else:
+            # Debugging tip: You can comment out the "with mp.Pool" block, and use the line below:
+            # self.result = [self.run(item) for item in iterable]
```

Comment:
This isn't needed if you set the number of processes to 1.

Reply:
Agreed, addressed with [doc, comment and logging updates](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/6d84efc018bb5a2d466c06caa56705e9531d2af7)